selenium: Remove webdriverio npm package

vaughnwalters opened the pull request:
selenium: Remove webdriverio npm package

The `webdriverio` package does not need to be an explicit dependency. It is a dependency of `@wdio/cli`.

Bug: https://phabricator.wikimedia.org/T325059

zeljkofilipin commented on package-lock.json:
```diff
@@ -1,3800 +1,12 @@
 {
 	"name": "wdio-wikibase",
-	"version": "5.1.0",
-	"lockfileVersion": 2,
```
You probably changed the lockfileVersion by mistake.

vaughnwalters replied:
Yes you are correct, I have reverted this now @zeljkofilipin

zeljkofilipin reviewed (changes requested):
Please revert the change to lockfileVersion.

vaughnwalters committed:
selenium: Remove webdriverio npm package

The `webdriverio` package does not need to be an explicit dependency.
It is a dependency of `@wdio/cli`.

Bug: T325059

zeljkofilipin commented:
To make sure this didn't break anything, we have to run Wikibase, WikibaseLexeme and wikibase-termbox test suites with this change. According to [codesearch](https://codesearch-beta.wmcloud.org/search/?q=wdio-wikibase), those repositores use this package.
